packages/core/src/parsers/ComponentParser.js

```diff
@@ -618,7 +631,10 @@ class ComponentParser {
     return node;
   }
 
-  render(content, filePath) {
+  render(file, content, cwf = file) {
```

Comment:
parameter order standardisation with `ComponentPreprocessor`, since there is also now the additional `cwf parameter